components/ui/TopNavigation/TopNavigation.tsx

```
@@ -0,0 +1,92 @@
+  title,
+  type = 'none',
+  onNavigationClick,
+  rightIcon,
+  onRightIconClick,
+  className,
+}: PresetTopNavigationProps) {
+  return (
+    <TopNavigationRoot className={className}>
+      <Left>
+        {type !== 'none' && (
+          <button onClick={onNavigationClick} className={styles.iconButton}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)